### PR TITLE
Add 20 new OD tests

### PR DESCRIPTION
**Pull request by anirudh711**
Found OD tests in the repository - https://github.com/json-iterator/java
Path to logs in VM- /home/ar80/idflakies-logs/json-iterator-java/dtfixingtools
More details - https://github.com/TestingResearchIllinois/idoft/issues/1102

### CODE REVIEW

**Comment by darko-marinov** [comment]
Are the tests `OD-Vic` or `OD-Brit`? If victims, did you find some polluter(s)? If you don't have all that info, add an issue with the test order (see older `OD` tests that added files such as `flaky-test-list.txt`).

**Comment by anirudh711** [comment]
I have an added an issue - https://github.com/TestingResearchIllinois/idoft/issues/1102 with more details.

**Comment by darko-marinov** [comment]
Thanks for more info. Please add https://github.com/TestingResearchIllinois/idoft/issues/1102 in `Notes`.

**Comment by anirudh711** [comment]
> Thanks for more info. Please add #1102 in `Notes`.

@darko-marinov I have updated pr-data.csv with the issue in Notes

File: pr-data.csv [hunk 1]
```diff
@@ -4051,7 +4051,10 @@ https://github.com/jReddit/jReddit,ac0b518f190c30228e401ef66d53329bbd27bad6,.,co
 https://github.com/jReddit/jReddit,ac0b518f190c30228e401ef66d53329bbd27bad6,.,com.github.jreddit.request.util.KeyValueFormatterTest.testFormatMultiple,ID,Opened,https://github.com/jReddit/jReddit/pull/158,
 https://github.com/jReddit/jReddit,ac0b518f190c30228e401ef66d53329bbd27bad6,.,com.github.jreddit.request.util.KeyValueFormatterTest.testFormatMultipleUTF8,ID,Opened,https://github.com/jReddit/jReddit/pull/158,
 https://github.com/jrtom/jung,1f579fe5d74ecbaecbe32ce6762e1fa9e17ed225,jung-algorithms,edu.uci.ics.jung.algorithms.shortestpath.TestMinimumSpanningTree.testDAG,ID,Opened,https://github.com/jrtom/jung/pull/258,
+https://github.com/json-iterator/java,6925cf4c19d313504b416f58a349a36bf563e0e1,.,com.jsoniter.extra.TestPreciseFloat.test_direct_encode,OD,,,
```
Comment (darko-marinov) [inline pr-data.csv:4054]: Where is the issue in the Notes?

Comment (anirudh711) [reply pr-data.csv:4054]: Sorry i had missed a few entries, added them now

**Pull request merged**